chapters/commonvalidity/build_acceleration_structure_common.txt

```diff
@@ -2,7 +2,7 @@
 // Common to acceleration structure build commands
 
   * [[VUID-{refpage}-pOffsetInfos-03402]]
-    Each element pname:ppOffsetInfos[i] must: be a valid pointer to an array of
+    Each element of pname:ppOffsetInfos[i] must: be a valid pointer to an array of
```

Comment:
@oddhack actually this was somewhat intentional. It is a single hypotetical element `array[i]`, not an array.